src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -38,14 +38,19 @@ void get_unsupported_operations_and_failures(const std::shared_ptr<Model>& model
     for (const auto& node : model->get_ordered_ops()) {
         if (const auto& fw_node = ov::as_type_ptr<FrameworkNode>(node)) {
             auto op_type = fw_node->get_decoder()->get_op_type();
+            // if this operation is encountered among unsupported operations
+            // or conversion failures, skip it
+            if (failures.count(op_type) > 0 ||
+                std::find(unsupported_operations.begin(), unsupported_operations.end(), op_type) !=
```

Comment:
please use set instead of vector.
If I remember correctly you said that in the tests the order of "error messages" is hardcoded, I believe you can easily fix the tests if you use search instead of the order.